contracts/modules/Checkpoint/EtherDividendCheckpoint.sol

```diff
@@ -211,7 +196,7 @@ contract EtherDividendCheckpoint is DividendCheckpoint {
         Dividend storage dividend = dividends[_dividendIndex];
         uint256 remainingWithheld = dividend.dividendWithheld.sub(dividend.dividendWithheldReclaimed);
         dividend.dividendWithheldReclaimed = dividend.dividendWithheld;
-        address owner = IOwnable(securityToken).owner();
+        address payable owner = IOwnable(securityToken).owner();

```

Comment:
Probably better to explicitly cast owner to payable here, and not make owner generally payable?

Reply:
Either way seems fine to me. Casting just looks ugly and takes a few more bytes though. To be honest, I don't even understand why they had to split address into payable and non-payable.

Let me know which way you prefer and I'll change it accordingly.